### PR TITLE
feat: [PIE-4903]: GA pipelines, remove /pipeline-list and move new UX under /pipelines

### PR DESCRIPTION
**Pull request by udayvunnam**
### Summary

- remove /pipeline-list and move new UX under /pipelines
- dead code removal with next PR

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

File: src/modules/70-pipeline/pages/pipeline-list/PipelineListTable/PipelineListCells.tsx [hunk 1]
```diff
@@ -256,15 +256,8 @@ export const MenuCell: CellType = ({ row, column }) => {
...
       >
-        <Button
-          minimal
-          icon="Options"
-          onClick={e => {
-            e.stopPropagation()
-            setMenuOpen(true)
-          }}
-        />
-        <Menu style={{ minWidth: 'unset', backgroundColor: 'unset' }} onClick={e => e.stopPropagation()}>
```
Comment (udayvunnam) [inline src/modules/70-pipeline/pages/pipeline-list/PipelineListTable/PipelineListCells.tsx:267]: stopPropagation() not required anymore anywhere in pipelines

**Comment by diptiman-adak** [comment]
Quality gate Quality gate: **_Failed_** 
 


            
 #### 3 Issues 
* Code issue 3 Code issues 

 [Details](https://app.embold.io/organization/gh/harness/repositories/e864badd0b95ab81c060b15b85f15701/pull-request/10640)

**Comment by udayvunnam** [comment]
retrigger titlecheck

File: src/modules/70-pipeline/pages/pipeline-list/PipelineListTable/PipelineListCells.tsx (hunk 1 above)
Comment (udayvunnam) [reply src/modules/70-pipeline/pages/pipeline-list/PipelineListTable/PipelineListCells.tsx:267]: ^^ @dhruv-harness

**Review by vkbansal-harness — DISMISSED** [review]

**Review by dhruvUpadhyay07 — APPROVED** [review]

**Review by vkbansal-harness — APPROVED** [review]

File: cypress/integration/70-pipeline/PipelinesList.spec.ts [hunk 2]
```diff
@@ -1,97 +1,54 @@
...
-
-      cy.contains('p', 'Stages').should('be.visible')
-      cy.contains('p', 'Stage 5').should('be.visible')
-      cy.contains('p', '+5').should('be.visible')
-
-      cy.contains('p', 'Build Repo').should('be.visible')
-      cy.contains('p', 'CCD').should('be.visible')
-
-      cy.contains('p', 'Services').should('be.visible')
-      cy.contains('p', 'Newdds').should('be.visible')
```
Comment (udayvunnam) [inline cypress/integration/70-pipeline/PipelinesList.spec.ts:47]: redundant assertions that are more suitable on jest integration tests, These have become obsolete with new UX

**Review by yc147 — APPROVED** [review]

**Pull request merged**